osbuild/buildroot.py

```diff
@@ -44,6 +46,16 @@ def stdout(self):
     def stderr(self):
         return self.output
 
+class StageTimeoutError(Exception):
+    """Exception raised for time out errors in creating images.
+    """
+
+    def __init__(self, message="rpm instalation timed out."):
```

Comment:
`instalation` that could have been me! The thing is, this can potentially happen in any stage, so I guess we probably shouldn't mention `rpm`. Also if we can't set anything else then a message we can just throw the standard `TimeoutError` (which this error should derive from).

Reply:
OK cool,

ill remove the custom error and call TimeoutError directly :smile: